src/sentry/feedback/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import BoundedBigIntegerField, Model, region_silo_only_model, sane_repr
+from sentry.db.models.fields import UUIDField
+from sentry.db.models.fields.array import ArrayField
+
+
+@region_silo_only_model
+class Feedback(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    project_id = BoundedBigIntegerField(db_index=True)
+    replay_id = models.CharField(max_length=32, null=True, db_index=True)
+    url = models.CharField(max_length=1000, null=True)
+    error_ids = ArrayField()
+    trace_ids = ArrayField()
```

Comment:
This might be dropped.

https://github.com/getsentry/sentry/pull/55566#discussion_r1317857527